content/docs/sidebar.json

```diff
@@ -523,75 +499,49 @@
     "type": "external"
   },
   {
-    "label": "CML",
     "slug": "cml",
```

Comment:
For some reason removing the "CML" label hides the nav entry @julieg18 . All other auto-upper-case seem to work fine. Probably related with the hardcoded redirect...

<img width="346" alt="image" src="https://user-images.githubusercontent.com/1477535/171346393-7a615193-9f95-4e7b-823d-d6703541ee90.png">

Reply:
Good to know, thanks (fixed in a74facb74).

> We should def. document the doc engine features at some point 😬 rel. https://github.com/iterative/gatsby-theme-iterative/issues/17